includes/Server.hpp

```diff
@@ -2,15 +2,9 @@
 # define CLASS_SERVER_HPP
 
 # include "Socket.hpp"
-# include <signal.h>
-# include <stdio.h>
-# include <cstring>
-# include <unistd.h>
-# include <time.h>
-# include <sys/wait.h>
-# include <stdlib.h>
+# include "utils.hpp"
 
-# define PORT "3490"
+# define MAXEVENTS 10
```

Comment:
If everything is in `utils.hpp` also this define might go there as well I guess - nothing major though :)